x-pack/plugins/session_view/public/components/ProcessTreeNode/styles.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEuiTheme } from '@elastic/eui';
+
+const TREE_INDENT = 32;
+
+interface StylesDeps {
+  depth: number;
+}
+
+export const useStyles = ({ depth }: StylesDeps) => {
```

Comment:
nice, I like this

Although I think this might break memoization, so maybe we should wrap it in useMemo, not a big deal but might catch us off guard in the future

Reply:
good call.